app/_hub/kong-inc/jwt/index.md

```diff
@@ -489,27 +509,31 @@ field in the token. Keys issued by Auth0 have their `iss` field set to
 validate the `iss` field for the `key` parameter when creating the
 Consumer.
 
-Send requests through, only tokens signed by Auth0 will work:
+Send requests through. Only tokens signed by Auth0 will work:
 
 ```bash
 $ curl -i http://localhost:8000 \
     -H "Host:example.com" \
-    -H "Authorization:Bearer {{TOKEN}}"
+    -H "Authorization:Bearer <TOKEN>"
 ```
 
 Success!
```

Comment:
This line doesn't actually add any value here... we're not showing output of the request, so how do we know it was successful? I'd remove it.

Reply:
> This line doesn't actually add any value here... we're not showing output of the request, so how do we know it was successful? I'd remove it.

Agree, also thought about that.

Reply:
Removed Success! all versions. We didn't even show any output.